engine/src/main/java/org/terasology/rendering/opengl/FrameBuffersManager.java

```diff
@@ -449,6 +463,15 @@ public void setPostProcessor(PostProcessor postProcessor) {
     }
 
     /**
+     * TODO: java doc
+     *
+     * @param bindFBO
+     */
+    public void subscribe(BindFBO bindFBO) {
```

Comment:
I think the signature should be:

`public void subscribe(FBOManagerSubscriber subscriber)`

No?